src/generated/graphql.tsx

```diff
@@ -0,0 +1,494 @@
+import { gql } from '@apollo/client';
```

Comment:
What was your rationale to also commit this file? Since it is auto-generated, I personally wouldn't commit it (and instead put it in gitignore).

Reply:
Sry, Accidently pushed it.